docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -533,7 +533,7 @@ p2p-port="1789"
 ```
 
 Specifies the port on which P2P peer discovery listens.
-The default is 30303.
+The default is 30303. The peer discovery port must be open on both TCP and UDP.
```

Comment:
I don't think I'd add these comments here - it assumes there's a firewall set up so its not really a descriptive comment about the config but assumes a certain networking environment.

Reply:
Added link to port configuration section